main.py

```diff
@@ -45,6 +46,17 @@
 DEBUG_GROUPED_BY_OWNERS = CONFIG['debug']['grouped_by_owners'].get(bool)
 DEBUG_FILTERED_BY_ORGS = CONFIG['debug']['filtered_by_org'].get(bool)
 
+@functions_framework.http
+def http_request(request):
+    now = dt.now()
+    date_value = dt.strftime(now, "%Y-%m-%dT%H:%M:%S.%fZ")
+    try:
+        message, status_code =  main()
+    except logging.exception as err:
+        message, status_code =  "Fail", 410
```

Comment:
Can't the actual return message and status code be used?
Can't you use some info from the exception itself?